lib/routes/user/connect_to_pay/payment_details_form.dart

```diff
@@ -74,8 +75,13 @@ class _PaymentDetailsFormState extends State<PaymentDetailsForm> {
                         currency: widget._account.currency,
                         controller: _amountController,
                         decoration: new InputDecoration(
-                            labelText: widget._account.currency.displayName +
-                                " Amount"),
+                          labelText: widget._account.currency.displayName + " Amount",
```

Comment:
Instead of adding this decoration for every instance of AmountField, it is better to implement it once inside the AmountFormField.